app/views/landing-page.html

```diff
@@ -1,4 +1,5 @@
 <default-header class="top-header"></default-header>
+<toast-notifications></toast-notifications>
```

Comment:
I spoke to @jwforres this morning, and we were both thinking we should just add this to index.html so it's available everywhere. I don't see a reason to wait to use toasts where it makes sense.